examples/fluids/README.md

```diff
@@ -326,21 +326,9 @@ For the Isentropic Vortex problem, the following additional command-line options
   - Strength of vortex < 10
   - `5`
   -
-
-* - `-euler_test`
-  - Euler test option (`test_0`-`test_4`)
-  - `none`
-  -
 :::
 
-This problem represents the exact solution to five test cases, which are selected by `-euler_test`, where
-- `test_0`: Isentropic vortex
-- `test_1`: Constant zero velocity, density constant, total energy constant
-- `test_2`: Constant nonzero velocity, density constant, total energy constant
-- `test_3`: Constant zero velocity, pressure constant, density and internal energy non-constant
-- `test_4`: Constant nonzero velocity, pressure constant, density and internal energy non-constant
-
-and it can be run with:
+This problem can be run with:
 
 ```
 ./navierstokes -problem euler_vortex -mean_velocity .5,-.8,0. -euler_test test_0
```

Comment:
I think this would become `-euler_test isentropic_vortex`, though that's kind of repetitive with `-problem euler_vortex`. If it's the default, then this option wouldn't need to be documented here. It can still be found via `-help`.

Reply:
Oops! Sorry, I missed that one! Just fixed it, thanks!